index.html

```diff
@@ -27,23 +25,26 @@ <h2 id="origin">Origin Story</h2>
           <h2 id="story">My Earlier Years</h2>
           <p> I later went to Rose Hill Jr. High and later Lake Washington High School. Later, I signed up and joined the Army in 2004. I served in three combat deployments to Iraq and Afghanistan. I served a total four years in Savannah, GA. While in the Army, I met my beautiful wife, Jennifer, in Savannah. Click below to learn more about the unit I was in! </p>
         </section>
-<div>
+
+        <div>
             <a href="https://www.army.mil/ranger/"><img src="https://s-media-cache-ak0.pinimg.com/736x/b9/8c/7e/b98c7ede6447f3003a119693026f3a0b.jpg"align="middle" width="25%" height="25%" frameborder="3" style="border:3"></a>
-          </div>
+        </div>
 
 
         <section>
           <h2 id="present">Present</h2>
-          <p> After my time in the Army, I came back to Washington with Jennifer. I was employed in social work for several years. During this time, we had two amazing girls, Avery 4-years-old, and Zoe 1-year-old. I finished my four year degree and decided I wanted to learn a new skill, software development. I quit my job and started at school at Code Fellows! <br />
-          </p>
-        </section>
-        <div>
-        <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2689.407117538558!2d-122.35402068436908!3d47.61821697918551!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x5490153758c88a61%3A0xe3d3fd29c55db518!2sCode+Fellows+-+Seattle+Campus!5e0!3m2!1sen!2sus!4v1488926687682" width="600" height="450" frameborder="3" style="border:3" allowfullscreen align="middle"></iframe></div><br/>
+            <p> After my time in the Army, I came back to Washington with Jennifer. I was employed in social work for several years. During this time, we had two amazing girls, Avery 4-years-old, and Zoe 1-year-old. I finished my four year degree and decided I wanted to learn a new skill, software development. I quit my job and started at school at Code Fellows! <br />
+            </p>
+          </section>
+          <div>
+            <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2689.407117538558!2d-122.35402068436908!3d47.61821697918551!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x5490153758c88a61%3A0xe3d3fd29c55db518!2sCode+Fellows+-+Seattle+Campus!5e0!3m2!1sen!2sus!4v1488926687682" width="600" height="450" frameborder="3" style="border:3" allowfullscreen align="middle"></iframe>
+          </div>
+          <br/>
 
-      <script type= "text/javascript" src="script.js">
+        <script type= "text/javascript" src="script.js">
```

Comment:
One issue you have here is that you are looking for a script.js file but you have an app.js file. So your JS did not show up. You just need to fix this before closing your pull request